src/wp-includes/widgets/class-wp-widget-media-video.php

```diff
@@ -228,17 +228,35 @@ public function render_control_template_scripts() {
 		?>
 		<script type="text/html" id="tmpl-wp-media-widget-video-preview">
 			<# if ( data.error && 'missing_attachment' === data.error ) { #>
-				<div class="notice notice-error notice-alt notice-missing-attachment">
-					<p><?php echo $this->l10n['missing_attachment']; ?></p>
-				</div>
+				<?php
+				wp_admin_notice(
+					$this->l10n['missing_attachment'],
+					array(
+						'type'               => 'error',
+						'additional_classes' => array( 'notice-alt', 'notice-missing-attachment' ),
+					)
+				);
+				?>
 			<# } else if ( data.error && 'unsupported_file_type' === data.error ) { #>
-				<div class="notice notice-error notice-alt notice-missing-attachment">
-					<p><?php echo $this->l10n['unsupported_file_type']; ?></p>
-				</div>
+				<?php
+				wp_admin_notice(
+					$this->l10n['unsupported_file_type'],
+					array(
+						'type'               => 'error',
+						'additional_classes' => array( 'notice-alt', 'notice-missing-attachment' ),
```

Comment:
I suspect this is an error, and this should have a different class. There's a JS event attached to links inside this class, though, so it might be correct, but should have a more abstract name. Not relevant to this PR, however.